randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -278,7 +282,7 @@ Extra - room_id: [6]
               # TODO: damage values
               Gravity Suit and Damage Runs (Beginner)
   > Other to Serris Escape
-      Can Climb High Jump Wall
+      Trivial
```

Comment:
nice. i assume it was this jump?
![grafik](https://github.com/user-attachments/assets/341298b2-4846-4c1f-9503-9b779081bf3b)

Reply:
correct